chat-plugins/trivia.js

```diff
@@ -418,27 +406,33 @@ var commands = {
 	// trivia game commands
 	new: function (target, room) {
 		if (room.id !== 'trivia' || !this.can('broadcast', null, room) || !target) return false;
-		Trivia.getInstance().startSignups(target, room, this);
+		if (!room.Trivia) room.Trivia = Trivia;
```

Comment:
Putting this in every command seems a bit excessive. Probably save yourself some work and just attach it to CommandParser, which is available all the time.